MANIFEST.in

```diff
@@ -3,5 +3,7 @@ include LICENSE
 include README.rst
 recursive-include cl_sii *py
 recursive-include cl_sii/data/cte/schemas-json *.schema.json
-recursive-include cl_sii/data/ref/factura_electronica/schemas-xml *.xsd
+recursive-include cl_sii/data/ref/factura_electronica/schemas-xml/2013_02_07_sii_official *.xsd
```

Comment:
@jtrh @glarrain I'm not entirely sure if these changes will be necessary, I know that `recursive-include` would recursively include all files, but will it keep the directory structure?